v0/destinations/canny/data/createPostConfig.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "destKey": "boardID",
+    "sourceKeys": [
+      "properties.boardID",
+      "properties.boardId",
+      "properties.board.id"
+    ],
+    "sourceFromGenericMap": false,
+    "required": true
+  },
+  {
+    "destKey": "byID",
+    "sourceKeys": [
+        "properties.byID",
+        "properties.byId",
+        "properties.by.id"
+      ],
+    "sourceFromGenericMap": false,
+    "required": false
+  },
+  {
+    "destKey": "categoryID",
+    "sourceKeys": [
+        "properties.categoryID",
+        "properties.categoryId",
+        "properties.category.id"
+      ],
+    "sourceFromGenericMap": false,
+    "required": false
+  },
+  {
+    "destKey": "customFields",
+    "sourceKeys": "properties.customFields",
```

Comment:
This map is not correct. We should put everything from properties after removing boardID, byID, categoryID,details, eta, etaPublic, title, imageURLs.

Reply:
User should create each and every customField on Canny's Dashboard and then should send them here, orelse the call will not happen. Won't it cause a problem?

Reply:
We will share these details in the doc. But let's not take it as `customFields` key.